src/ApiService/ApiService/ServiceConfiguration.cs

```diff
@@ -26,7 +26,9 @@ public interface IServiceConfig {
 
     public string? DiagnosticsAzureBlobContainerSasUrl { get; }
     public string? DiagnosticsAzureBlobRetentionDays { get; }
-
+    public string? CliAppId { get; }
```

Comment:
Do these need to be nullable?

Reply:
Yup, just based-off how the service config is set up. All of the values are nullable, despite the fact that they are mostly all present all of the time.